### PR TITLE
Fix widgets hover

### PR DESCRIPTION
**Pull request by ivanmalagon**
⚠️ This PR is a copy of https://github.com/CartoDB/deep-insights.js/pull/652 Moved here since DI is deprecated ⚠️ 

Related to: https://github.com/CartoDB/deep-insights.js/issues/640

### CODE REVIEW

File: app/assets/stylesheets/deep-insights/themes/scss/widgets/_progress-bar.scss [hunk 1]
```diff
@@ -8,8 +8,8 @@
   margin: $baseSize / 2 0;
   border-radius: $baseSize / 2;
 }
+
 .CDB-Widget-progressState {
-  // @include css3-prefix(animation, state-progress 600ms ease-out);
```
Comment (matallo) [inline app/assets/stylesheets/deep-insights/themes/scss/widgets/_progress-bar.scss]: 🔥

File: app/assets/stylesheets/deep-insights/themes/scss/widgets/_progress-bar.scss [hunk 2]
```diff
@@ -18,9 +18,20 @@
...
+    width: 100%;
+    height: 100%;
+    transition: all 250ms ease-in-out;
+    background: transparent;
+    content: '';
+  }
 }
+
 .is-disabled .CDB-Widget-progressState {
-  // @include css3-prefix(animation, state-progress 0 ease-out);
```
Comment (matallo) [inline app/assets/stylesheets/deep-insights/themes/scss/widgets/_progress-bar.scss]: 🔥

File: app/assets/stylesheets/deep-insights/themes/scss/widgets/_list.scss [hunk 3]
```diff
@@ -87,7 +87,14 @@
 .CDB-Widget-listItemInner {
   margin: 0 $sMargin-section;
   padding: 4px 0;
+
+  &:hover {
+    .CDB-Widget-progressState::before {
+      background: rgba(0, 0, 0, 0.16);
```
Comment (matallo) [inline app/assets/stylesheets/deep-insights/themes/scss/widgets/_list.scss]: `#000` ?

**Comment by matallo** [comment]
shall we add specific specs for hover?

**Comment by piensaenpixel** [comment]
like a charm

**Pull request merged**